tools/sync-diff-inspector.md

```diff
@@ -18,7 +18,7 @@ sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一
 
 GitHub 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
 
-下载地址：[sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz)
+下载地址：[tidb-latest-linux-amd64](https://download.pingcap.org/tidb-latest-linux-amd64.tar.gz)
```

Comment:
Any description about we have added sync_diff_inspector into TiDB? Then, the user can know they do not need to download it again.

Reply:
@csuzhangxc I mentioned that in this docs PR about Tools Download page: https://github.com/pingcap/docs/pull/855

Reply:
@lilin90 in https://github.com/pingcap/docs/pull/855, is *This **enterprise tools** package includes all the following tools:*?

Reply:
As far as I know, Yes. @csuzhangxc

Reply:
so, whether needing to update description here, because it's *TiDB* not *TiDB Enterprise Tools*?

Reply:
or, can we tell the user to get *sync-diff-inspector* from *tidb-enterprise-tools*?

Reply:
@GregoryIan What's your opinion, from the _TiDB_ package or the _tidb-enterprise-tools_ package?

Reply:
I will change to tidb-enterprise-tools, this contains all the tools.